src/passes/TupleOptimization.cpp

```diff
@@ -320,8 +320,13 @@ struct TupleOptimization : public WalkerPass<PostWalker<TupleOptimization>> {
         // we were confused earlier and the target should not be.
         assert(sourceBase);
 
+        // The source and target may have different lane types due to subtyping
```

Comment:
Usually we use "element" rather than "lane" to describe tuple elements. ("lane" is specific to SIMD vectors.)

Reply:
Thanks, fixed. Also fixed some existing appearances.